boards/qemu/rx/qemu_rx.yaml

```diff
@@ -9,7 +9,15 @@ simulation:
 arch: rx
 toolchain:
   - cross-compile
+  - zephyr
+  - zephyr-gnu
```

Comment:
zephyr-gnu is not needed and is not used anymore in the sdk, just drop it for now.